includes/Admin.php

```diff
@@ -82,4 +84,87 @@ public static function admin_body_class( $class ) {
 
 		return $class;
 	}
+
+	/**
+	 * Pre-fills post content with a web-story/embed block.
+	 *
+	 * @param string $content Default post content.
+	 *
+	 * @return string Pre-filled post content if applicable, or the default content otherwise.
+	 */
+	public static function prefill_post_content( $content ) {
+		$embed_story = isset( $_GET['from-web-story'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+
+		if ( ! $embed_story ) {
+			return $content;
+		}
+
+		$post_id = absint( $_GET['from-web-story'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
For VIP GO, I have had to do this before.
```suggestion
		$post_id = absint( sanitize_text_field( wp_unslash( $_GET['from-web-story'] ) ) ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Reply:
That's really unnecessary for a number though. It will never contain slashes.

If anything I'd add an `is_numeric` check and bail early if it's not a number.

Reply:
This example may seem unnecessary but I have had a very similar line rejected by the VIP GO team, as it is not secure. I even copied this validation for a project. It is overly careful, but the lints on VIP GO are very unforgiving.

Reply:
Good to know! Well then I'll add'em back.